mmv1/third_party/terraform/services/compute/resource_compute_disk_test.go.erb

```diff
@@ -1720,3 +1720,40 @@ resource "google_compute_disk" "foobar" {
 }
 `, diskName, storagePoolUrl)
 }
+
+func TestAccComputeDisk_accessModeSpecified(t *testing.T) {
+	t.Parallel()
+
+	diskName := fmt.Sprintf("tf-test-disk-accessmode-%s", acctest.RandString(t, 10))
+	accessMode := "READ_ONLY_MANY"
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config:    testAccComputeDisk_accessModeSpecified(diskName, accessMode),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_compute_disk.foobar", "access_mode", accessMode),
+				),
+			},
+			{
+				ResourceName:      "google_compute_disk.foobar",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
```

Comment:
Can you add more test steps for this new field to test if updating works or not?

Doc: [how to add a update test](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test)

Reply:
added